native/cocos/renderer/pipeline/RenderAdditiveLightQueue.cpp

```diff
@@ -94,8 +95,9 @@ void RenderAdditiveLightQueue::recordCommandBuffer(gfx::Device *device, scene::C
         const auto &dynamicOffsets = lightBatchedPass.dynamicOffsets;
         for (size_t i = 0; i < dynamicOffsets.size(); ++i) {
             const auto lightIdx = lights[i];
+            const auto *light = _validPunctualLights[lightIdx];
```

Comment:
Should change `AdditiveLightPass::lights` to `vector<Light*>` too?

Reply:
sure